README.md

```diff
@@ -150,42 +153,15 @@ all dependencies are managed by Nix.
 ### Build using Stack
 
 1. Install [Stack](https://docs.haskellstack.org/en/stable/README/).
-2. Make sure you have the following system libraries installed:
+2. Make sure you have the libraries listed below installed. This list
+   might be incomplete.
```

Comment:
That's the complete set of native deps.

Reply:
Now it its. But what if somebody forgets to update readme?